### DISCUSSION TITLE
谷歌同步有点不懂

### DISCUSSION THREAD

**Discussion by fuhuojie123** [immersive-translate/immersive-translate#2110] [Help]
### 概述 | Summary

![Screenshot_2024-09-16-08-20-41-140_com mmbox xbrowser](https://github.com/user-attachments/assets/9cd9e5ae-1ed2-4636-83c1-19481b980f6a)
![Screenshot_2024-09-16-08-20-45-504_com mmbox xbrowser](https://github.com/user-attachments/assets/e2542fd8-96e3-4086-b91d-a2c91d249990)
手动备份节点的导入，是从云端导入到本地吗。
那Googledrive，是从本地导入到云端还是云端导入到本地。

**Comment by theowenyoung** [comment]
> 那Googledrive，是从本地导入到云端还是云端导入到本地。


点击按钮 是自动判断 本地和云端谁该覆盖谁，根据最后更新时间。
